posts/MarkoB/2023-09-05-hibernate-search-7-0-0-Beta1.adoc

```diff
@@ -0,0 +1,158 @@
+= Hibernate Search 7.0.0.Beta1 is out
+Marko Bekhta
+:awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+:hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/7.0/reference/en-US/html_single/
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-version-family: 7.0
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 32169
+---
+
+We just published Hibernate Search 7.0.0.Beta1,
+a first beta release of the next major version of Hibernate Search.
+
+The main theme of this new major version is upgrades:
+JDK 11 as a baseline,
+switch to Jakarta EE and to the latest Hibernate ORM,
+upgrade of the Lucene backend to the next major version (Lucene 9).
+
+Beyond that, 7.0.0.Beta1 also
+includes experimental compatibility with Amazon OpenSearch Serverless,
+publishes a Hibernate Search BOM containing all of its public artifacts,
+drops compatibility with EOL'd Elasticsearch versions 5.x/6.x,
```

Comment:
This may not be something we want to brag about in the headline :)

```suggestion
drops compatibility with EOL'd Elasticsearch versions 5.x/6.x,
```

Reply:
😏 😆

Reply:
Oops, my suggestion was wrong. Updated my comment: I meant to suggest removing this.

Reply:
And you did it, thanks :)